### ISSUE TITLE
Keyboard focus in sidebar can't be returned from chat

### ISSUE THREAD

**Issue by Remu-rin** [telegramdesktop/tdesktop#24212]
### Steps to reproduce

1. Open some chat and sidebar, choose e.g. Photos
2. Try `page down`/`page up` on keyboard
3. Click on chat (e.g. share button, text, open image)
4. Try to return focus to sidebar to continue scroll with keyboard

### Expected behaviour

There should be a way to return keyboard focus to sidebar without loosing state (e.g. you scrolled down several weeks/months).

### Actual behaviour

There is no way to return keyboard focus to sidebar without loosing scroll state. Seems that keyboard focus is changed to sidebar only when you click to open category (Photos, files, etc). If you lose keyboard focus to something else, nothing can return it back to sidebar: clicks in different places, opening photo, selecting items, forwarding.
The only way to return keyboard focus is to exit and enter category again, but you lose state you were in and need to scroll down again. And without #17245 it's frustrating.

### Operating system

Windows 10

### Version of Telegram Desktop

3.6.1

### Installation source

Static binary from official website

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
Still relevant. But now (at least on Linux) focus is locked on keyboard from the start and it's impossible to page up/down in Photos in step 2.

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
Still relevant.

**Comment by Remu-rin** [comment]
With recent implementation of https://github.com/telegramdesktop/tdesktop/issues/17245 (Calendar) sidebar focus for page up/down is not that important.
Closing.
